ibis/expr/types.py

```diff
@@ -421,6 +421,21 @@ class ScalarExpr(ValueExpr):
     def _type_display(self):
         return str(self.type())
 
+    def to_projection(self):
+        """
+        Promote this column expression to a table projection
+        """
+        roots = self._root_tables()
+        if len(roots) > 1:
+            raise com.RelationError(
+                'Cannot convert array expression '
```

Comment:
Should this be `scalar expression` since this method is defined on `ScalarExpr`?

Reply:
bump

Reply:
Updated.